frontend/src/Map/Markers.js

```diff
@@ -5,7 +5,7 @@ export default function Markers({ stations }) {
   return (
     <div>
       {stations
-        .filter(
+        ?.filter(
```

Comment:
De akkor itt ki is lehet venni a filtert…

Reply:
Javítva

Reply:
visszajött a színe is pedig hozzá se nyúltam

Reply:
Parádés, akkor most már akár mergelhetjük is :)